src/systemTest/java/com/flightstats/hub/model/ChannelConfigExpirationSettings.java

```diff
@@ -21,53 +21,36 @@
         return Optional.ofNullable(mutableTime);
     }
 
-    public static class ChannelConfigExpirationSettingsBuilder {
-        private boolean keepForever;
-        private long ttlDays;
-        private DateTime mutableTime;
-        private long maxItems;
-
-        public ChannelConfigExpirationSettingsBuilder keepForever(boolean keepForever) {
-            this.keepForever = keepForever;
-            validate();
-            return this;
-        }
-
-        public ChannelConfigExpirationSettingsBuilder ttlDays(long ttlDays) {
-            this.ttlDays = ttlDays;
-            validate();
-            return this;
-        }
-
-        public ChannelConfigExpirationSettingsBuilder mutableTime(DateTime mutableTime) {
-            this.mutableTime = mutableTime;
-            validate();
-            return this;
-        }
+    public static Builder builder() {
+        return new Builder();
+    }
 
-        public ChannelConfigExpirationSettingsBuilder maxItems(long maxItems) {
-            this.maxItems = maxItems;
-            validate();
-            return this;
+    public static class Builder extends ChannelConfigExpirationSettingsBuilder {
```

Comment:
You shouldn't have to do an extends. You should just be able to do this:
```
    public static class ChannelConfigExpirationSettingsBuilder {
        public ChannelConfigExpirationSettings build() {
            validate(); // Could have a validate() on the builder or on the class. I'd put it on the builder.
            return new ChannelConfigExpirationSettings(foo, bar, baz);
        }
  
        private void validate() {
            ...validate foo, bar, and baz
        }
    }
```
If I need to replace the Builder generated code, I'll usually use IntelliJ's Refactor->Delombok to generate the code, then strip out everything except what I want to replace. Lombok is (usually) smart enough to generate everything it doesn't see you've already implemented.

Reply:
When I don't do an extends, but I keep @Builder on there, it complains that `build()` is already defined. At that point if I want to avoid an extends, I have to delombokify what seems like most of the builder behavior. :-/

Reply:
I'll mess with it a bit to see if I'm missing something.

Reply:
...it's a test class and I think I'm done fighting with lombok for today. I'll make another go at it if I touch it again.

Reply:
Here's an example that I think does what you're looking for:
```
@Value
@NoArgsConstructor(force = true)
@AllArgsConstructor
@Builder
public class Test {
    String a;
    String b;

    public static void main(String[] args) {
        Test t = Test.builder().b("bar").build();
        System.out.println(t);
    }
    static class TestBuilder {
        public Test build() {
            validate();
            return new Test(this.a, this.b);
        }

        private void validate() {
            if (a == null) throw new RuntimeException("Hey, you have to set a!");
        }
    }
}
```